django_app_lti/views.py

```diff
@@ -1,17 +1,19 @@
+from future import standard_library
+standard_library.install_aliases()
 from django.http import HttpResponse
 from django.shortcuts import redirect
-from django.core.urlresolvers import reverse
+from django.urls import reverse
 from django.contrib.auth import logout
 from django.views.generic import View
 from django.conf import settings
 
-from ims_lti_py.tool_config import ToolConfig
+from lti import ToolConfig
 from braces.views import CsrfExemptMixin, LoginRequiredMixin
 
 from .models import LTIResource, LTICourse, LTICourseUser
 
-import urlparse
-import urllib
+import urllib.parse
+import urllib.request, urllib.parse, urllib.error
```

Comment:
Looks like you are importing `urllib.parse` twice here. I think it may be more idiomatic to import like the following:
`from urllib import error, parse, request` and then updating the resulting calls to those functions?

Reply:
Good catch. I have fixed it. Thanks.